students/10th/yeajinpark/login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+      <div class = "background" >
+        <img src = "img/logo_text.png">
+
+        <form class = "input-wrap">
```

Comment:
```suggestion
        <img src="img/logo_text.png">
        <form class="input-wrap">
```

불필요한 줄바꿈은 없애주세요. 속성과 속성값 사이의 공백도 없애주시는게 좋겠습니다.

Reply:
넵 공백 딜리트하겠습니다.